src/testing/puppeteer/puppeteer-screenshot.ts

```diff
@@ -123,6 +118,11 @@ export async function pageCompareScreenshot(
     }
   }
 
+  const screenshotOpts = createPuppeteerScreenshotOptions(opts, width, height);
```

Comment:
Can we add a small comment here that clarifies that we just updated `width` and `height`, and that the order of operations is important here (since `opts.clip` can influence `width` and `height` in the conditional directly above this)